client/src/components/interfaces/Comment.ts

```diff
@@ -0,0 +1,9 @@
+export interface Comment {
+  commentID?: string;
+  postID?: string;
+  userID?: string;
+  comment: string;
+  createAt?: Date;
+  updatedAt?: Date;
+  likes?: number;
+}
```

Comment:
If the `Comment` and `Post` interfaces are meant to be used in multiple places, I recommend putting them in `src/types` folder, and add a `index.ts` in the the same folder that exports them so you can `import { Comment } from "types"`. See `src/pages/index` for something similar.